java/src/jmri/web/servlet/directory/DirectoryHandler.java

```diff
@@ -37,7 +37,6 @@ public DirectoryHandler() {
         this.setWelcomeFiles(new String[]{"index.html"}); // NOI18N
     }
 
-    @Override
```

Comment:
While this change allows you to compile, this doDirectory method is not called since it no longer overrides anything, and this breaks the web server, so merely removing this annotation to allow compilation means this branch should fail tests.

Reply:
Interestingly, test results with this override removed and the newest version of jetty installed are the same as with the older version of jetty installed.  Passed in appveyor failed on both Travis jobs with a timeout in ZeroConf.  I'm just going to revert this particular change and look for other potential causes.